### DISCUSSION TITLE
Docx Validation error: The element has invalid child element http://schemas.openxmlformats.org/wordprocessingml/2006/main:rtl

### DISCUSSION THREAD

**Discussion by lucasonni** [OfficeDev/Open-XML-SDK#1747] [General]
Hi, attached is the file that is not validated, with the following error:

_The element has invalid child element http://schemas.openxmlformats.org/wordprocessingml/2006/main:rtl_

Below is the code:

> Stream fileStream = new MemoryStream(Convert.FromBase64String(documento.StringBase64));
> string errorDescription = "";
> WordprocessingDocument? wordprocessingDocument = null;
> int count = 0;
> try
> {
>     wordprocessingDocument = WordprocessingDocument.Open(fileStream, false);
>     OpenXmlValidator validator = new OpenXmlValidator(DocumentFormat.OpenXml.FileFormatVersions.Microsoft365);
>     foreach (ValidationErrorInfo error in validator.Validate(wordprocessingDocument))
>     {
>         count++;
>         if (count == 1)
>             errorDescription = "Description: " + error.Description;
>         else
>             errorDescription += error.Description;
>     }
> }
> catch (Exception ex)
> {
>     count++;
>     errorDescription = ex.Message;
> }
> finally
> {
>     if(wordprocessingDocument is not null)
>         wordprocessingDocument.Dispose();
> }

I also tried with the other supported versions of the Word format, from Office2007 to Office2021, with the same error.

Can anyone help me understand?

Thanks!
Luca

[TextControlDoc.docx](https://github.com/user-attachments/files/16088624/TextControlDoc.docx)

**Comment by twsouthwick** [comment]
Where is this file coming from? Sounds like it has invalid markup - do you have a reason to believe it is wrong?
